_posts/2025-06-30-pyos-scipy-2025.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: single
+title: "pyOpenSci at SciPy 2025 - science, Python and Community"
+excerpt: "pyOpenSci is excited to be at SciPy 2025 this year! Find us at our second packaging tutorial, a birds of a feather session and hanging out in sessions where our community members are leading events and talks. Don't forget to join us for our beginner-friendly sprint after the meeting!"
+author: "pyopensci"
+permalink: /blog/pyopensci-at-scipy-2025.html
```

Comment:
For each post we have to generate a link like this - normally it's `blog/post-name-here.html` considering SEO when we post